data/companies/alma/corporate.yaml

```
@@ -0,0 +1,22 @@
+---
+type: corporate
+name: Alma Media
+template: publisher
+comment: methodology question - if we are excluding print emissions, how do we exclude print employees?
```

```suggestion
public_identifier: alma
comment: methodology question - if we are excluding print emissions, how do we exclude print employees?
```

From [utils/public_yaml_files.py](https://github.com/scope3data/methodology/blob/main/scope3_methodology/utils/public_yaml_files.py#L30) the `public_identifier` attribute looks to be needed.